### ISSUE TITLE
Fall einreichen: Hinweise setzen, wenn Eingaben fehlen

### ISSUE THREAD

**Issue by Th3da** [DetektivKollektiv/angular-ui#171]
Falls vergessen wurde

-  einen Medientyp auszuwählen
-  die Nutzungsbedingungen etc. zu bestätigen
-  (leeres Textfeld?)
 
sollte beim Klicken auf den Submit Button ein entsprechender Hinweis kommen  - am besten mit Cursor-Position auf dem fehlenden Eiintrag.

**Comment by ChristianBay** [comment]
Soweit ich es sehe, kann ich erst auf den Submit Button klicken wenn die ganzen Bedingungen die du oben auflistest erfüllt sind (davor ist der Button inaktiv).
Meinst du wir sollen den Workflow ändern, d.h. ich kann vorher schon Submit klicken und dann kommen die Validierungsfehler?

Ich persönlich finde es okay wie es ist. Durch das '*' an den Feldnamen erkennt der gewiefte Internetbenutzer, dass es Pflichtfelder sind. Ein Erklärungstext zum * haben wir aktuell allerdings noch nicht auf der Seite, daher würde ich vorschlagen "nur" den Erklärungstext sinnvoll im Formular zu platzieren.

**Comment by Th3da** [comment]
Nein, auf keinen Fall den Workflow ändern. Und klar, für den gewieften User ist das alles kein Thema.
Meine, dass wir auch keinen Erklärungstext für den Stern brauchen, das wird auch jeder kennen.

Es geht nur um eine Komfort-Funktion, da das Formular so lang ist.
Wenn ich unten beim Submit Button bin, seh ich nicht, was fehlt. Ich glaube, es ist oben auch nicht markiert. 
Ich muss jedenfalls erstmal wieder hochscrollen und schauen, wo ich was übersehen habe.
Ganz doof ist, wenn ich die Auswahl des Medientyps vergessen habe.
Da komm ich vielleicht nicht sofort drauf.

Komfortabel für den User wäre, wenn der Pointer zu der Stelle springt, wo was fehlt, ggf. mit Hinweistext.

Aber das Thema hat keinerlei Prio im Moment.
